socs/agents/http_camera/agent.py

```diff
@@ -200,6 +224,8 @@ def add_agent_args(parser=None):
     pgroup = parser.add_argument_group("Agent Options")
     pgroup.add_argument("--config-file", type=str, help="Config file path relative to OCS_CONFIG_DIR")
     pgroup.add_argument("--mode", choices=['acq', 'test'])
+    pgroup.add_argument("--renew-token", type=int, default=2700,
+                        help="Renew API token after this amount of seconds. Used for Reolink cameras.")
```

Comment:
It seems to me like this should go into the `cameras.yaml` config file.

It's unique to the Reolink cameras. You can still default to 2700, but that also lets you set it per camera if needed for whatever reason. Existing configs would still work since it'll just grab the default if missing.

Maybe there's another reason to have it global like this?

Reply:
Right, I decided to go this route to future-proof in case we add other brands that also require tokens, which I imagine wouldn't be uncommon.

Reply:
Alright, I don't think having it in the config file is any less future proof, but we can roll with this if you'd like. I'll go ahead and merge now.